x/mint/client/cli/query.go

```diff
@@ -116,3 +117,31 @@ func GetCmdQueryAnnualProvisions() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmqQueryTargetSupply implements a command to return the current target supply value.
+func GetCmqQueryTargetSupply() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "target-supply",
+		Short: "Query the current target supply for this phase value",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := types.NewQueryClient(clientCtx)
+
+			params := &types.QueryTargetSupplyRequest{}
+			res, err := queryClient.TargetSupply(cmd.Context(), params)
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintString(fmt.Sprintf("%s\n", res.TargetSupply))
```

Comment:
I think we should put it as the denom here too in the query

ex: `140015400000` is not as explicit as `{"denom":"ujuno","amount":"140015400000"}`